assn2_AioHTTP/web_app_aiohttp.py

```diff
@@ -0,0 +1,40 @@
+import zipfile
+from aiohttp import web
+
+
+async def index():
+    return web.Response(text=
+    """
+    <!DOCTYPE html>
+    <head>
+    <title>Web app for extracting zip files</title>
+    </head>
+    <body>
+    <form action="/extract" enctype="multipart/form-data" method="post">
+    <input type="file" name="zip_file">
+    <input type="submit">
+    </form>
+    </body>
+    </html>
+    """, content_type='text/html')
+
+
+async def extract(request):
+    data = await request.post()
+    zip_file = data['zip_file'].file
+    with zipfile.ZipFile(zip_file,'r') as z:
+        extracted_files = z.namelist()
+        html = "<ul>"
```

Comment:
Avoid having html script in python script. For this you can have separate html files.